### PR TITLE
implement long and short for the price overview for the athlete page

### PR DESCRIPTION
**Pull request by pearlson16**
# Description
implement long and short for the price overview for the athlete page

Fixes Jira Ticket # https://athletex.atlassian.net/browse/AX-417

## Type of change
Please delete options that are not relevant.
- [x] New feature (non-breaking change which adds functionality)

image
Screenshot_1649440136

# How Has This Been Tested?
test on multiple devices 

# Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have assigned 2 reviewers to check my work
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings

### CODE REVIEW

**Review by ketonkss4 — APPROVED** [review]

File: lib/pages/AthletePage.dart [hunk 1]
```diff
@@ -45,12 +53,69 @@ class _AthletePageState extends State<AthletePage> {
   Widget build(BuildContext context) {
     if (listView == 1) return DesktopScout();
 
-    return kIsWeb.kIsWeb ? buildWebView(context) : buildMobileView(context);
+    return kIsWeb.kIsWeb ? buildWebViewContainer(context) : buildMobileView(context);
+  }
+
+  IndexedStack buildWebViewContainer(BuildContext context) {
+
+    var longMarketPrice = "4.18 AX";
```
Comment (ketonkss4) [inline lib/pages/AthletePage.dart:61]: make all these vars finals if they arent gonna be changing

File: lib/pages/AthletePage.dart [hunk 2]
```diff
@@ -1264,15 +1691,15 @@ class _AthletePageState extends State<AthletePage> {
...
+                                textStyle(Colors.white, 24, false, false))),
                         Divider(thickness: 1, color: greyTextColor),
                         Row(
                             mainAxisAlignment: MainAxisAlignment.spaceBetween,
                             children: <Widget>[
                               Text("Sport / League",
                                   style: textStyle(
                                       greyTextColor, 20, false, false)),
-                              Text("American Football / NFL",
+                              Text("BaseBall / MLB",
```
Comment (ketonkss4) [inline lib/pages/AthletePage.dart:1702]: Use the sport property in the athlete model

File: lib/pages/AthletePage.dart [hunk 3]
```diff
@@ -122,792 +255,970 @@ class _AthletePageState extends State<AthletePage> {
...
+                                              color: greyTextColor),
+                                          Row(
+                                              mainAxisAlignment: MainAxisAlignment
+                                                  .spaceBetween,
+                                              children: <Widget>[
+                                                Text("Sport / League",
+                                                    style: textStyle(
+                                                        greyTextColor, 12,
+                                                        false, false)),
+                                                Text("MLB",
```
Comment (ketonkss4) [inline lib/pages/AthletePage.dart:832]: use athlete.sport.name

**Review by ketonkss4 — CHANGES_REQUESTED** [review]

File: lib/pages/AthletePage.dart (hunk 1 above)
Comment (pearlson16) [reply lib/pages/AthletePage.dart:61]: done

File: lib/pages/AthletePage.dart (hunk 2 above)
Comment (pearlson16) [reply lib/pages/AthletePage.dart:1702]: done

**Comment by ketonkss4** [comment]
Make the Chart Book value two decimal places

File: lib/pages/AthletePage.dart (hunk 3 above)
Comment (pearlson16) [reply lib/pages/AthletePage.dart:832]: done

**Review by user2745 — APPROVED** [review]

**Review by ketonkss4 — APPROVED** [review]

**Review by Friendlygeckos — APPROVED** [review]

**Pull request merged**